cfme/cloud/tenant.py

```diff
@@ -55,7 +54,8 @@ class TenantEntities(View):
     title = Text('//div[@id="main-content"]//h1')
     table = Table('//div[@id="list_grid"]//table')
     search = View.nested(Search)
-    # element attributes changed from id to class in upstream-fine+, capture both with locator
+    # element attributes changed from id to class in upstream-fine+, capture
+    # both with locator
```

Comment:
Why is this split on 2 lines? Its only 94 chars.  I'd rather see the word 'locator' removed than split this if I'm counting wrong and it is over 100.